### PR TITLE
BCH cosmetic changes

### PR DESCRIPTION
**Pull request by tahmeedshah**
https://jira.cbd.int/browse/DEVDESK-682
https://jira.cbd.int/browse/DEVDESK-700
https://jira.cbd.int/browse/DEVDESK-702

### CODE REVIEW

File: app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js [hunk 1]
```diff
@@ -79,9 +79,14 @@ import 'services/main';
...
 				}
-                $scope.setTab = function () {
-                    $scope.tab = 'edit';
-                };
+                $scope.setTab = function ($event) {
+					$scope.tab = 'edit';
+					var href = $($event.target).attr("href");
+					$timeout(function (  ) {
+					 $('.pagination li a[href!="'+href+'"]').closest('li').removeClass('active');
+					 $('.pagination li a[href="'+href+'"]').closest('li').addClass('active');
```
Comment (blaisefonseca) [inline app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js:87]: can u use element, so the lookup is only on the directive and not the whole page

File: app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js [hunk 2]
```diff
@@ -79,9 +79,14 @@ import 'services/main';
 					return $scope.onBuildDocumentSelectorQuery(queryOptions);
 		
 				}
-                $scope.setTab = function () {
-                    $scope.tab = 'edit';
-                };
+                $scope.setTab = function ($event) {
+					$scope.tab = 'edit';
+					var href = $($event.target).attr("href");
+					$timeout(function (  ) {
```
Comment (blaisefonseca) [inline app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js:85]: u can use $scope.asynApply((){})

File: app/views/forms/edit/bch/directives/edit-risk-assessment.directive.html [hunk 3]
```diff
@@ -192,21 +192,21 @@
...
 														<div km-yes-no ng-model="document.isForCommercialUse" locales="document.header.languages"></div>
 														<div ng-if="document.isForCommercialUse" style="padding-left:50px;">
-															<div class="control-group" style="margin-top: 10px; margin-bottom: 5px;">
+															<div class="inner-block">
 																Should this decision be forwarded to the OECD Secretariat for possible inclusion in the
 															<a rel="noopener" translation-url href="https://biotrackproductdatabase.oecd.org/" target="_blank"> BioTrack Product Database</a>?
 															</div>
 															<div class="km-yes-no" name="forwardToOECD" ng-model="document.forwardToOECD"></div>
-															<div class="control-group" style="margin-top: 10px; margin-bottom: 5px;">
+															<div class="inner-block">
```
Comment (blaisefonseca) [inline app/views/forms/edit/bch/directives/edit-risk-assessment.directive.html:200]: why inner-block this should be control-group to match the app label?

Comment (tahmeedshah) [reply app/views/forms/edit/bch/directives/edit-risk-assessment.directive.html:200]: inner-block has extra styles required for inner questions, like margin and font size.
https://jira.cbd.int/browse/DEVDESK-700

File: app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js (hunk 2 above)
Comment (tahmeedshah) [reply app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js:85]: to remove all active tabs and then activate the current tab we used $timeout,
Using $scope.asynApply  will only change active tabs and the page will not change.

File: app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js (hunk 1 above)
Comment (tahmeedshah) [reply app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js]: updated to $element

File: app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js [hunk 4]
```diff
@@ -79,9 +79,14 @@ import 'services/main';
...
+						$element.find('.pagination li a[href!="'+href+'"]').closest('li').removeClass('active');
+						$element.find('.pagination li a[href="'+href+'"]').closest('li').addClass('active');
+					},100);
```
Comment (blaisefonseca) [inline app/views/forms/edit/bch/directives/edit-biosafety-expert.directive.js:88]:
```suggestion
					var href = $element.find($event.target).attr("href");
					$scope.$applyAsync(function (  ) {
						$element.find('.pagination li a[href!="'+href+'"]').closest('li').removeClass('active');
						$element.find('.pagination li a[href="'+href+'"]').closest('li').addClass('active');
					});
```

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**